Fix NAT option handling

rmounce opened the pull request:
Regression introduced with b119882

Don't overwrite CAKE_FLOW_NAT_FLAG in flow_mode with TCA_CAKE_FLOW_MODE.

tohojo commented:
The trouble is that since we can abort when reading the nat mode, it kinda needs to be first, or we risk partially applying options. That's why it was moved to the top...

rmounce committed:
Fix NAT option handling

Regression introduced with b1198824af267ac98a237b7a607e6bdf46aa168d

Don't overwrite CAKE_FLOW_NAT_FLAG in flow_mode with TCA_CAKE_FLOW_MODE.

Signed-off-by: Ryan Mounce <ryan@mounce.com.au>

rmounce commented:
I see... my updated commit should do the trick then.

tohojo commented:
Ryan Mounce <notifications@github.com> writes:

> I see... my updated commit should do the trick then.

Yeah, that's better; thanks! :)

-Toke
